{
    "type": "bundle",
    "id": "bundle--e1a5d05f-200e-4889-acf5-66e2dcf13543",
    "spec_version": "2.0",
    "objects": [
        {
            "type": "relationship",
            "id": "relationship--576f0fb4-4cd7-467b-9208-3977321705e4",
            "created": "2022-09-29T19:56:25.812Z",
            "revoked": false,
            "external_references": [
                {
                    "source_name": "DFIR Conti Bazar Nov 2021",
                    "description": "DFIR Report. (2021, November 29). CONTInuing the Bazar Ransomware Story. Retrieved September 29, 2022.",
                    "url": "https://thedfirreport.com/2021/11/29/continuing-the-bazar-ransomware-story/"
                }
            ],
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "modified": "2022-09-29T19:56:25.812Z",
            "description": "During [C0015](https://attack.mitre.org/campaigns/C0015), the threat actors obtained files and data from the compromised network.(Citation: DFIR Conti Bazar Nov 2021)",
            "relationship_type": "uses",
            "source_ref": "campaign--78068e68-4124-4243-b6f4-76e4e5be8a06",
            "target_ref": "attack-pattern--3c4a2599-71ee-4405-ba1e-0e28414b4bc5",
            "x_mitre_deprecated": false,
            "x_mitre_version": "0.1",
            "x_mitre_attack_spec_version": "2.1.0",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5"
        }
    ]
}